probe-dictionary/explore.js

```diff
@@ -431,24 +431,34 @@ function shortVersion(v) {
   return v.split(".")[0];
 }
 
-function friendlyRecordingRange(firstVersion, expiry) {
-  if (expiry == "never") {
-    return `from ${firstVersion}`;
+function friendlyRecordingRangeForHistory(history, channel, filterPrerelease) {
+  const last = array => array[array.length - 1];
+
+  if ((channel == "release") && filterPrerelease) {
```

Comment:
comment nit: why is this fixup needed? That's for bridging the optout/optin to the prerelease/release definitions, right?

Reply:
On the "release" channel we only want to account for the release probes. The most readable approach seems to just filter out the history parts we don't want to account for.

This is only used for the detail view though; in the search results view we want different behavior - hence the `filterPrerelease` flag.